WalletWasabi/WabiSabi/Client/CoinJoinCoinSelector.cs

```diff
@@ -63,8 +70,7 @@ public static class CoinJoinCoinSelector
 
 		if (semiPrivateCoins.Length + redCoins.Length == 0)
 		{
-			// Let's not mess up the logs when this function gets called many times.
-			return ImmutableList<TCoin>.Empty;
+			throw new CoinJoinClientException(CoinjoinError.AllCoinsPrivate, "No red coins or semi private coins in the wallet.");
```

Comment:
This is not true. It is possible that the round parameters are not allowing us to select proper coins.

Reply:
Add LogInfo => No coins suitable for mixing for this round.